### PR TITLE
Presentation of the Home Day Picker

### PR DESCRIPTION
**Pull request by MKiperszmid**
Day Picker Preview

### CODE REVIEW

File: app/src/main/java/com/mk/tasky/agenda/home/presentation/components/HomeDayPickerItem.kt [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+@Composable
+fun HomeDayPickerItem(
+    day: LocalDateTime,
+    isSelected: Boolean = false,
+    onDayClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Column(
+        modifier = modifier.clip(RoundedCornerShape(100))
+            .background(if (isSelected) Orange else Color.Transparent).padding(8.dp).clickable {
```
Comment (philipplackner) [inline app/src/main/java/com/mk/tasky/agenda/home/presentation/components/HomeDayPickerItem.kt]: If you apply the clickable modifier before the padding, the click ripple effect will match the shape :)

File: app/src/main/java/com/mk/tasky/core/presentation/TaskyBackground.kt [hunk 2]
```diff
@@ -18,31 +18,46 @@ import com.mk.tasky.ui.theme.White
 
 @Composable
 fun TaskyBackground(
-    @StringRes title: Int,
+    @StringRes title: Int? = null,
```
Comment (philipplackner) [inline app/src/main/java/com/mk/tasky/core/presentation/TaskyBackground.kt]: Naming recommendation: `titleResId`, else it'll sound like a string

**Review by philipplackner — APPROVED** [review]

File: app/src/main/java/com/mk/tasky/core/presentation/TaskyBackground.kt (hunk 2 above)
Comment (MKiperszmid) [reply app/src/main/java/com/mk/tasky/core/presentation/TaskyBackground.kt]: Awesome. Fixed!

File: app/src/main/java/com/mk/tasky/agenda/home/presentation/components/HomeDayPickerItem.kt (hunk 1 above)
Comment (MKiperszmid) [reply app/src/main/java/com/mk/tasky/agenda/home/presentation/components/HomeDayPickerItem.kt]: Right! Fixed

**Pull request merged**